diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 9f93f95..0c6513d 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1,9 +1,9 @@
 add_definitions(-DBASE_LOG_NAMESPACE=${PROJECT_NAME})
 
-rock_find_qt5(Core REQUIRED)
+rock_find_qt5(Core Widgets REQUIRED)
 
 rock_library(vizkit3d_plugin_information
              HEADERS Vizkit3dPluginInformation.hpp
              SOURCES Vizkit3dPluginInformation.cpp
-             DEPS_PKGCONFIG vizkit3d base-lib 
-             LIBS Qt5::Core)    
+             DEPS_PKGCONFIG vizkit3d base-lib
+             LIBS Qt5::Core Qt5::Widgets)
